app/models/message.rb

```diff
@@ -0,0 +1,7 @@
+class Message < ActiveRecord::Base
+
+  belongs_to :user
+
+  validates :content, presence: true
+  validates :content, length: { minimum: 3 }
```

Comment:
The two lines can be reduced into one

`validates :content, presence: true, length: { minimum: 3 }`